optimum/habana/diffusers/__init__.py

```diff
@@ -1,5 +1,8 @@
 from .pipelines.auto_pipeline import AutoPipelineForInpainting, AutoPipelineForText2Image
 from .pipelines.controlnet.pipeline_controlnet import GaudiStableDiffusionControlNetPipeline
+from .pipelines.controlnet.pipeline_stable_video_diffusion_controlnet import (
+    GaudiStableVideoDiffusionPipelineControlNet,
```

Comment:
```suggestion
    GaudiStableVideoDiffusionControlNetPipeline,
```

just a personal suggestion to be compliant with other naming like `GaudiStableDiffusionControlNetPipeline`

Reply:
Done, thanks.